### ISSUE TITLE
About the code

### ISSUE THREAD

**Issue by hhBonnie** [ChangxinWang/BoFiCap#1]
Thank you very much for sharing the code. Would you consider writing a more detailed readme.md? Looking forward to your reply!

**Comment by ChangxinWang** [comment]
Hello, I have received your feedback and thank you very much for your attention!!  I will add a user manual this weekend, including basic operations such as training and prediction. I will gradually add more detailed things in the future (if the update is slow, I'm very sorry because I'm a bit busy recently for job hunting😭

**Comment by hhBonnie** [comment]
> Hello, I have received your feedback and thank you very much for your attention!! I will add a user manual this weekend, including basic operations such as training and prediction. I will gradually add more detailed things in the future (if the update is slow, I'm very sorry because I'm a bit busy recently for job hunting😭
Thank you so much for your quick reply! Look forward to your future revisions. And good luck in your job hunting!

**Comment by hhBonnie** [comment]
您好呀，方便给一下data下的文件吗？

**Comment by ChangxinWang** [comment]
Of course. And I'm sorry about that I forgot to upload my new readme.md😭. The preprocess data will be uploaded together soon. I see you are chinese friend, so if you have any other detailed question, you can also contact me via my wechat 18551709301
